Modules/xASL_module_ASL.m

```diff
@@ -81,6 +81,13 @@
     end
 end
 
+% Check if Hadamard data
+if isfield(x,'HadamardType') 
```

Comment:
This was a misunderstanding :) I mean that you simply define x.HadamardType = 0 if missing, not a new HadamardType variable :)

Reply:
Having a duplicitous variable would make stuff confusing, and will also not be visible elsewhere ;)